conf/test.config

```diff
@@ -71,19 +71,20 @@ profiles {
         params.no_intervals          = true
     }
     targeted {
-        params.intervals           = "${baseDir}/tests/csv/some_other_muli.bed"
+        params.intervals           = "${baseDir}/tests/reference/multi_intervals_chr21_small.bed"
+        params.wes                 = true
     }
     tools {
         params.input               = "${baseDir}/tests/csv/3.0/recalibrated.csv"
         params.dbsnp               = "${params.genomes_base}/data/genomics/homo_sapiens/genome/chr21/germlineresources/dbsnp_138.hg38.vcf.gz"
         params.fasta               = "${params.genomes_base}/data/genomics/homo_sapiens/genome/chr21/sequence/genome.fasta"
         params.germline_resource   = "${params.genomes_base}/data/genomics/homo_sapiens/genome/chr21/germlineresources/gnomAD.r2.1.1.vcf.gz"
-        params.intervals           = "${baseDir}/tests/csv/multi_intervals.bed"
+        params.intervals           = "${baseDir}/tests/reference/multi_intervals_chr21_large.bed"
```

Comment:
Shouldn't that be in test-datasets actually?
I'm ok to have the csvs here in the sarek repo, but I think the references and/or the data should be in the test-datasets repo, no?

Reply:
sure we can push it there.